hs-bindgen/src-internal/HsBindgen/Frontend/Pass/Parse/Type/DeclId.hs

```diff
@@ -0,0 +1,83 @@
+module HsBindgen.Frontend.Pass.Parse.Type.DeclId (
```

Comment:
Before implementing #807 , I used `DeclId` in the predicate `match` function. To avoid circular module dependencies, I created a separate module for `DeclId`. Then, I moved on to using `QualName`, and this separate module is not strictly necessary anymore. However, it can stand on its own, so I left it for now.